### PR TITLE
Feature/rankings

### PR DESCRIPTION
**Pull request by Trinkes**

### CODE REVIEW

File: 2048/2048/src/main/java/com/appcoins/eskills2048/rankins/RankingsPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        return StatisticsTimeFrame.WEEK;
+      case 2:
+        return StatisticsTimeFrame.ALL_TIME;
+      default:
+        throw new InvalidParameterException("Invalid position " + position);
+    }
+  }
+
+  @Override public int getItemCount() {
+    return 3;
```
Comment (nnelas) [inline 2048/2048/src/main/java/com/appcoins/eskills2048/rankins/RankingsPagerAdapter.java:37]: this 3 is related to how many items are inside StatisticsTimeFrame, right? why just not: StatisticsTimeFrame.values().length

File: 2048/2048/src/main/res/layout/player_rank_layout.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        android:layout_height="match_parent"
+        android:layout_margin="8dp">
+
+        <TextView
+            android:id="@+id/rankingRank"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="10dp"
+            app:layout_constraintEnd_toStartOf="@id/rankingUsername"
+            tools:text="1"
```
Comment (nnelas) [inline 2048/2048/src/main/res/layout/player_rank_layout.xml:27]: what's the difference between using tools:text and android:text ?

File: 2048/2048/src/main/res/layout/start_new_game_layout.xml [hunk 3]
```diff
@@ -54,8 +51,23 @@
...
+        <Button
+            android:id="@+id/goToPlayerStats"
+            style="@style/ButtonStyle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="32dp"
+            android:layout_marginTop="32dp"
+            android:layout_marginEnd="32dp"
+            android:layout_marginBottom="32dp"
+            android:text="[DEBUG] Get to player stats"
```
Comment (nnelas) [inline 2048/2048/src/main/res/layout/start_new_game_layout.xml:65]: is this debug on purpose? also, could you add this to strings.xml?

File: 2048/2048/src/main/java/com/appcoins/eskills2048/rankins/RankingsAdapter.java [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+import androidx.recyclerview.widget.RecyclerView;
+import com.appcoins.eskills2048.R;
+import com.appcoins.eskills2048.model.RankingsItem;
+import com.appcoins.eskills2048.model.RankingsTitle;
+import com.appcoins.eskills2048.model.UserRankingsItem;
+import java.util.List;
+
+public class RankingsAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+
+  private static final String TAG = "RankingsAdapter";
```
[the remark this answers is not in the document]
Comment (Trinkes) [reply 2048/2048/src/main/java/com/appcoins/eskills2048/rankins/RankingsAdapter.java:19]: removed

File: 2048/2048/src/main/res/layout/player_rank_layout.xml (hunk 2 above)
Comment (Trinkes) [reply 2048/2048/src/main/res/layout/player_rank_layout.xml:27]: When you use tools name space, it will never be shown in runtime

**Pull request merged**